config/initializers/friendly_id.rb

```diff
@@ -0,0 +1,107 @@
+# FriendlyId Global Configuration
+#
+# Use this to set up shared configuration options for your entire application.
```

Comment:
For studying reasons, you can keep these comments. But have in mind that when working on a real project you should remove the unnecessary comments.